src/patternfly/components/Popover/popover-arrow.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-popover__arrow{{#if popover-body--modifier}} {{popover-body--modifier}}{{/if}}"
+  {{#if popover-body--attribute}}
+    {{{popover-body--attribute}}} 
+  {{/if}}>
+  {{> @partial-block}}
```

Comment:
Do we need to include {{> @partial-block}} here? I'm not sure. I was wondering this when I was looking at how we want to set our rules. It's not hurting anything, so I don't know if it matters. What do you think?

Reply:
I don't see the harm in adding the standard `--modifier`, `--attribute` and `@partial-block` hbs code to all partials. I didn't have it initially, but it is more consistent to add it to any partial, and even though it doesn't do anything at the moment, it's there if we want to use it in the future. And doesn't add any bloat to PF itself, just a few lines of code to the workspace.